ReactAndroid/src/main/third-party/android/support/v4/BUCK

```diff
@@ -1,54 +1,131 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
 
 fb_native.android_library(
     name = "lib-support-v4",
     visibility = ["PUBLIC"],
     exported_deps = [
-        ":android-lifecycle-common",
-        ":android-lifecycle-core",
-        ":android-lifecycle-runtime",
-        ":android-lifecycle-viewmodel",
-        ":lib-support-v4-support-compat",
-        ":lib-support-v4-support-core-ui",
-        ":lib-support-v4-support-core-utils",
-        ":lib-support-v4-support-fragment",
+        ":animated-vector-drawable",
+        ":arch-common",
+        ":arch-runtime",
+        ":asynclayoutinflater",
+        ":collections",
+        ":coordinatorlayout",
+        ":cursoradapter",
+        ":customview",
+        ":documentfile",
+        ":drawerlayout",
+        ":interpolator",
+        ":support-compat",
+        ":support-core-ui",
+        ":support-core-utils",
+        ":support-vector-drawable",
+        ":support-fragment",
         ":lib-support-v4-support-media-compat",
+        ":lifecycle-common",
+        ":lifecycle-runtime",
+        ":lifecycle-viewmodel",
+        ":livedata-core",
+        ":livedata",
+        ":loader",
+        ":localbroadcastmanager",
+        ":print",
+        ":slidingpanelayout",
+        ":swiperefreshlayout",
+        ":versionedparcelable",
+        ":viewpager",
+        react_native_dep("third-party/android/support-annotations:android-support-annotations"),
     ],
 )
 
```

Comment:
can you add?

fb_native.prebuilt_jar(
    name = "android-support-annotations",
    binary_jar = ":support-annotations-binary.jar",
)